release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2784,6 +2784,30 @@ mount: /var/lib/kubelet/plugins/kubernetes.io/local-volume/mounts/local-pv-bc42d
 To workaround the issue, delete the `cloud-event-proxy-store-storage-class-http-events` `PVC` CR and re-deploy the PTP Operator.
 (link:https://issues.redhat.com/browse/OCPBUGS-12358[*OCPBUGS-12358*])
 
+[id="ocp-4-13-ran-known-issues-post-ga"]
+* During {ztp-first} provisioning of a {sno} managed cluster with secure boot enabled in the `SiteConfig` CR, multiple `ProvisioningError` errors are reported for the `BareMetalHost` CR during host provisioning.
+The error indicates that the secure boot setting is successfully applied in the Baseboard Management Controller (BMC), but the host is not powered on after the `BareMetalHost` CR is applied.
+To workaround this issue, perform the following steps:
++
+. Reboot the host.
+This ensures that the {ztp} pipeline applies the secure boot setting.
+
+. Restart {ztp} provisioning of the cluster with the same configuration.
+
++
+(link:https://issues.redhat.com/browse/OCPBUGS-8434[(*OCPBUGS-8434*)])
+
+* After installing a dual-stack {ztp} hub cluster and enabling dual-stack Virtual IP addresses (VIPs) and enabling the `virtualMediaViaExternalNetwork` flag in a `Provisioning` CR, the `IRONIC_EXTERNAL_URL_V6` environment variable incorrectly gets assigned an IPv4 address.
+(link:https://issues.redhat.com/browse/OCPBUGS-4248[(*OCPBUGS-4248*)])
+
+ZT servers have the `BiosRegistry` language set to `en-US` instead of `en`.
```

Comment:
@chadcrum Can you review/lgtm on this known issue text for https://issues.redhat.com/browse/OCPBUGS-4388 please.